doc-content/drools-docs/src/main/asciidoc/DMN/dmn-model-creating-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='dmn-model-creating-proc_{context}']
 = Creating and editing DMN models in {CENTRAL}
 
-You can use the DMN designer in {CENTRAL} to design DMN decision requirements diagrams (DRDs) and define decision logic for a complete and functional DMN decision model. {PRODUCT} provides design and runtime support for DMN 1.2 models at conformance level 3, and includes enhancements and fixes to FEEL and DMN model components to optimize the experience of implementing DMN decision services with {PRODUCT}. {PRODUCT} also provides runtime-only support for DMN 1.1 models at conformance level 3, but any DMN 1.1 models that you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models.
+You can use the DMN designer in {CENTRAL} to design DMN decision requirements diagrams (DRDs) and define decision logic for a complete and functional DMN decision model. {PRODUCT} provides design and runtime support for DMN 1.2 models at conformance level 3, and includes enhancements and fixes to FEEL and DMN model components to optimize the experience of implementing DMN decision services with {PRODUCT}. {PRODUCT} also provides runtime-only support for DMN 1.1 and 1.3 models at conformance level 3, but any DMN 1.1 and 1.3 models that you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models.
```

Comment:
It is not possible to open 1.3 models in the editor; so this paragraph is not quite right.

I guess something like
> {PRODUCT} also provides runtime-only support for DMN 1.1 and 1.3 models at conformance level 3, but any DMN 1.1 models that you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models. Support for DMN 1.3 in DMN designer in {CENTRAL} is not supported.

WDYT?

Reply:
@manstis, 
Have changed the paragraph as:

{PRODUCT} provides design and runtime support for DMN 1.2 models at conformance level 3, and runtime-only support for DMN 1.1 and 1.3 models at conformance level 3. You can design your DMN models directly in {CENTRAL} or import existing DMN models into your {PRODUCT} projects for deployment and execution. Any DMN 1.1 models that you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models. DMN 1.3 in DMN designer is not supported in {CENTRAL}.

Please let me know if you have any suggestions.